lib/Doctrine/Common/DataFixtures/Executor/ORMExecutor.php

```diff
@@ -4,74 +4,14 @@
 
 namespace Doctrine\Common\DataFixtures\Executor;
 
-use Doctrine\Common\DataFixtures\Event\Listener\ORMReferenceListener;
-use Doctrine\Common\DataFixtures\Purger\ORMPurgerInterface;
-use Doctrine\Common\DataFixtures\ReferenceRepository;
-use Doctrine\ORM\Decorator\EntityManagerDecorator;
-use Doctrine\ORM\EntityManager;
 use Doctrine\ORM\EntityManagerInterface;
 
 /**
  * Class responsible for executing data fixtures.
```

Comment:
Nitpicking but do we want to add `@final` here? To not introduce BC while giving the intention

Reply:
I have no strong oppinion here. But let's not do this in the scope of this PR.